jspwiki-util/src/main/java/org/apache/wiki/util/TextUtil.java

```diff
@@ -232,8 +232,11 @@ public static String replaceString( final String orig, final String src, final S
         if ( orig == null ) {
             return null;
         }
-        if ( src == null || dest == null ) {
-            throw new NullPointerException();
+        if ( src == null ) {
+            throw new IllegalArgumentException ( "src must not be null!" );
```

Comment:
similar to above comment, perhaps is better to throw a `NullPointerException` informing which field should not be `null`?